examples/sevensegment_demo.py

```diff
@@ -68,6 +69,11 @@ def show_message_alt(seg, msg, delay=0.1):
 def main():
     # create seven segment device
     device = get_device()
+
+    if not hasattr(device, 'segment_mapper'):
```

Comment:
Hmmm -  thinking about it, this check would be better off in the `sevensegment` class instead